move_param_node:
  ros__parameters:
    waypoints: ["init", "tv", "bathroom1", "bathroom2", "bedroom1", "bedroom2", "kitchen", "table"]
    waypoint_coords:
      init: [0.0, 0.0, 0.0]
      bathroom1: [-1.85, 3.0, 0.0]
      bathroom2: [-4.5, -0.6, 0.0]
      bedroom1: [-4.6, 1.75, 0.0]
      bedroom2: [-4.0, -4.0, 0.0]
      kitchen: [1.68, -2.6, 0.0]
      tv: [3.5, -0.5, 0.0]
      table: [4.5, 3.0, 0.0]
    rooms_colors:
      bathroom1: "red"
      bathroom2: "blue"
      bedroom1: "green"
      bedroom2: "red"
      kitchen: "green"
      tv: "blue"
      table: "red"
    colors_values:
      red: [0.0, 100.0, 0.0, 8.0, 255.0, 255.0]
      blue: [96.0, 108.0, 0.0, 164.0, 255.0, 255.0]
      green: [45.0, 114.0, 119.0, 84.0, 255.0, 255.0]
    objects:
      red: "cereals"
      blue: "ball"
      green: "apple"
